docs/samples/python/index.md

```diff
@@ -6,5 +6,6 @@ Presidio service can be used as python packages inside python scripts
 
 ## Table of contents
 
-1. [Jupiter notebook](https://github.com/microsoft/presidio/blob/main/docs/samples/python/presidio_notebook.ipynb)
-2. [Remote Recognizer](https://github.com/microsoft/presidio/blob/main/docs/samples/python/example_remote_recognizer.py)
+1. [Simple analysis and anonymization](https://github.com/microsoft/presidio/blob/main/docs/samples/python/presidio_notebook.ipynb)
```

Comment:
```suggestion
1. [Simple analysis and anonymization](presidio_notebook.ipynb)
```

Reply:
This PR also introduces mkdocs-jupyter, so we don't have to point out to the external notebook on GH. It will be rendered within mkdocs. Same for the previous notebook we have there (simple scenario).